### PR TITLE
add friends-app

### PR DESCRIPTION
**Pull request by beta-version-profile**
### Friends App

[Demo](http://obeisant-love.surge.sh) |
[Code base](https://github.com/beta-version-profile/frontend-2019-homeworks/tree/friends-app/submissions/beta-version-profile/friends_app)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by beta-version-profile** [comment]
@OleksiyRudenko, thank you for your comment. I'm not strong in style)

File: submissions/beta-version-profile/friends_app/app.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+const content = document.querySelector('#content')
+
+const state = {
+  data: [],
+  filters: {
+    search: null,
+    gender: null,
+    sortBy: null
+  }
+}
```
Comment (zonzujiro) [inline submissions/beta-version-profile/friends_app/app.js]: Can you describe me, why you have chosen such an approach?

File: submissions/beta-version-profile/friends_app/app.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+const isEqual = (a, b) => a === b
```
Comment (zonzujiro) [inline submissions/beta-version-profile/friends_app/app.js]: What for? :) IMO, redundant.

Comment (beta-version-profile) [reply submissions/beta-version-profile/friends_app/app.js]: So the record looks shorter and more elegant. I thought I would use more times in the code.

File: submissions/beta-version-profile/friends_app/app.js (hunk 1 above)
Comment (beta-version-profile) [reply submissions/beta-version-profile/friends_app/app.js]: It seemed to me convenient to save the state of the application in a separate object,

**Comment by zonzujiro** [comment]
Hey! You did a great job worth special recognition.

Here is your achievement badge:
[Kottans-Student-Course-progress-5%3D-Course%20Top--5%20performer-green.svg)](https://github.com/kottans/kottans/blob/master/endorsements.md)

Congratulations on the skills level-up!

**Comment by beta-version-profile** [comment]
Hey. For me it is very important, I will try to keep the level :)

**Pull request merged**